newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -130,6 +140,125 @@ const PrivateAssetPackInformationPage = ({
       assetPack => assetPack.id === privateAssetPackListingData.id
     );
 
+  const packsIncludedInBundleTiles = React.useMemo(
+    () => {
+      if (!assetPack || !privateAssetPackListingDatas) return null;
+
+      const includedPackIds =
+        privateAssetPackListingData.includedListableProductIds;
+      if (!includedPackIds) return null;
+
+      return includedPackIds.map(includedPackId => {
+        const includedAssetPackListingData = privateAssetPackListingDatas.find(
+          privatePackListingData => privatePackListingData.id === includedPackId
+        );
+        if (!includedAssetPackListingData) {
+          console.warn(`Included pack ${includedPackId} not found`);
+          return null;
+        }
+
+        const isPackOwned =
+          !!receivedAssetPacks &&
+          !!receivedAssetPacks.find(
+            pack => pack.id === includedAssetPackListingData.id
+          );
+        return (
+          <PrivateAssetPackTile
+            assetPackListingData={includedAssetPackListingData}
+            key={includedAssetPackListingData.id}
+            onSelect={() => onAssetPackOpen(includedAssetPackListingData)}
+            owned={isPackOwned}
+          />
+        );
+      });
+    },
+    [
+      assetPack,
+      privateAssetPackListingDatas,
+      receivedAssetPacks,
+      onAssetPackOpen,
+      privateAssetPackListingData,
+    ]
+  );
+
+  const bundlesNotOwnedContainingPackTiles = React.useMemo(
+    () => {
+      if (!assetPack || !privateAssetPackListingDatas) return null;
+
+      console.log(privateAssetPackListingDatas);
+      console.log(assetPack);
+
+      const bundlesContainingPack = privateAssetPackListingDatas.filter(
+        privatePackListingData =>
+          privatePackListingData.includedListableProductIds &&
+          privatePackListingData.includedListableProductIds.includes(
+            assetPack.id
+          )
+      );
+
+      console.log(bundlesContainingPack);
+
+      if (!bundlesContainingPack.length) return null;
+
+      return bundlesContainingPack
+        .map(bundleContainingPack => {
+          const isBundleOwned =
+            !!receivedAssetPacks &&
+            !!receivedAssetPacks.find(
+              pack => pack.id === bundleContainingPack.id
+            );
+          return !isBundleOwned ? (
+            <PromoBundleAssetPackTile
+              assetPackListingData={bundleContainingPack}
+              onSelect={() => onAssetPackOpen(bundleContainingPack)}
+            />
+          ) : null;
```

Comment:
I think you should display a message saying that they already own this pack through the bundle